src/items/item-persistence.js

```diff
@@ -602,6 +602,45 @@ class ItemPersistence {
     return _stateOrNull(PersistenceExtensions.averageBetween(this.rawItem, ...arguments));
   }
 
+  /**
+   * Gets the median value of the state of a given Item since a certain point in time.
+   *
+   * @example
+   * var yesterday = new Date(new Date().getTime() - (24 * 60 * 60 * 1000));
```

Comment:
Rather use JS-Joda in the example - we never use the native JS Date because it is not that great:

```suggestion
   * var yesterday = time.toZDT().minusDays(1);
```

Reply:
Changed